src/dotnet/Telemetry.cs

```diff
@@ -31,12 +42,26 @@ public Telemetry()
                 return;
             }
 
+            int sampleRate = ReciprocalSampleRateValue;
+            bool isCITestMachine = Env.GetEnvironmentVariableAsBool(ContinousIntegrationFlag);
```

Comment:
On who's CI machines?  Our customers?  Or the CI machines that build https://github.com/dotnet/cli ?